bin/update-releases.sh

```diff
@@ -56,25 +73,42 @@ do
 done
 
 update_submodule diego-release "${DIEGO_RELEASE}" src
-update_submodule cflinuxfs2-rootfs-release "${CFLINUXFS2_ROOTFS_RELEASE}" src
+update_submodule cflinuxfs2-rootfs-release "${CFLINUXFS2_OLD_ROOTFS_RELEASE}" src
+update_submodule cflinuxfs2-release "${CFLINUXFS2_ROOTFS_RELEASE}" src
 update_submodule garden-runc-release "${GARDEN_RUNC_RELEASE}" src
 
 CF_RELEASE_VERSION_INFO=$(curl --silent "https://api.github.com/repos/cloudfoundry/cf-release/contents/src?ref=${CF_RELEASE}")
 
+echo > ${GIT_ROOT}/_work/CF_RELEASE_VERSION_INFO "${CF_RELEASE_VERSION_INFO}"
+echo > ${GIT_ROOT}/_work/SUBMODULE_REFERENCES    ""
+
 get_submodule_ref () {
 	version_info=${1}
 	release=${2}
+
+	case $release in
+	    *-buildpack-*)
+		where=$(cd ${GIT_ROOT}/src/buildpacks/$release ; git remote -v | grep fetch | awk '{ print $2 }' | uniq)
```

Comment:
You can use the `-C` flag in git to do this without `cd`ing.

`git -C "${GIT_ROOT}/src/buildpacks/$release" remote -v`

IMO a little more obvious, plus one less command being executed.